crates/formality-types/src/grammar/ty/debug_impls.rs

```diff
@@ -32,6 +32,25 @@ impl Debug for RigidTy {
                     write!(f, "()")
                 }
             }
+            RigidName::FnDef(name) => {
+                let parameters = PrettyParameters::new("<", ">", parameters);
+                write!(f, "{name:?}{parameters:?}",)
```

Comment:
I'm surprised this is not `fn {name:?}`...?

Reply:
hmm, fair point, I would also expect `fn {name}`